enclave-runtime/src/lib.rs

```diff
@@ -785,7 +788,7 @@ where
 
 			// Notify watching clients of InSidechainBlock
 			let block = signed_block.block();
-			top_pool.on_block_created(block.signed_top_hashes(), block.hash());
+			author.on_block_created(block.signed_top_hashes(), block.hash());
```

Comment:
Use the author API instead of the top pool. So on this top-level we route everything through the author API. Instead of having a mix between author API and top pool directly.